pages/Guides/supernode-program/docs.en.md

```diff
@@ -8,90 +8,89 @@ taxonomy:
 
 ## About Supernodes
 
-The NEM Supernode's program is funded with XEM set aside during the Nemesis block. These awards are then given to high performance nodes for helping to secure the network. These nodes form a backbone of support for [light wallets](http://nem.ghost.io/lightwallet/), mobile wallets, and 3rd party apps so that users of these services might have access to the network that is easy, fast, and reliable without having to sync a blockchain by themselves or use untrustworthy centralized services.
+The NEM Supernode's Program is funded with XEM set aside during the Nemesis block. These rewards are then given to high performance nodes for helping to secure the network. These nodes form a backbone of support for light wallets, mobile wallets, and 3rd party apps so that users of these services might have access to the network that is easy, fast, and reliable without having to sync a blockchain by themselves or use untrustworthy centralized services.
```

Comment:
Change to:

"NEM is a cryptocurrency with 100% issuance in the Genesis block. In lieu of block rewards, 10,000,000 XEM has been set aside to reward validators: a full node that participates in the consensus and security of the network, and provides high-speed data availability for applications on the NEM blockchain."

Reply:
I'd be inclined to lead with emphasis on SuperNode Program rather than the chain / token issuance.. i.e.:

"The NEM Supernode Program rewards high performance validators that participate in the consensus and security of the network, and provide high-speed data availability for applications on the NEM blockchain. NEM's native token XEM was 100% issued in the Genesis block; in lieu of block reward, 10,000,000 XEM have been set aside for the SuperNode Program."

Phrasing is just a suggestion.